topo_processor/stac/asset_key.py

```diff
@@ -0,0 +1,6 @@
+from enum import Enum
+
+
+class AssetKey(Enum):
```

Comment:
`class AssetKey(str, Enum):`
See the [doc](https://docs.python.org/3/library/enum.html#others)